tests/execution/add-field-index-list.md

```diff
@@ -1,16 +1,16 @@
 # Sending field index list
 
 ```graphql @server
-schema {
+schema @server @upstream {
```

Comment:
We shouldn't need empty `@server` and `@upstream`.

Reply:
Without this the identity checks will fail.

Reply:
Need to fix it in the config.rs